docs/en_US/SearchSpaceSpec.md

```diff
@@ -92,3 +92,5 @@ Note that In Grid Search Tuner, for users' convenience, the definition of `qunif
 * Type 'qloguniform' behaves like 'quniform' except that it will first change the range to [log(low), log(high)] and sample and then change the sampled value back.
 
 Note that Metis Tuner only support numerical `choice` now
+
```

Comment:
> Note the Only Random Search/TPE/Anneal/Evolution tuner support nested search sapce. Fow now, we don't support showing nested search space in WebUI.

Suggest change to:

**Known Limitations**:
- Only Random Search/TPE/Anneal/Evolution tuner supports nested search space;
- We do not support nested search space ["Hyper Parameter" parallel graph](https://github.com/microsoft/nni/blob/master/docs/en_US/WebUI.md#view-hyper-parameter) now, the enhancement is being considered in #1110 , any suggestions or discussions or contributions are warmly welcomed.